nifi-nar-bundles/nifi-kudu-bundle/nifi-kudu-processors/src/main/java/org/apache/nifi/processors/kudu/AbstractKudu.java

```diff
@@ -84,6 +86,14 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    protected static final PropertyDescriptor INSERT_OPERATION = new PropertyDescriptor.Builder()
+            .name("INSERT OPERATION")
+            .description("Specify operation for this processor. Insert-Ignore will ignore duplicated rows")
+            .allowableValues("Insert", "Insert-Ignore", "Upsert")
```

Comment:
It's generally best practice for us to use an Enum instead of a list of strings when using `allowableValues`..it mainly just lends itself to cleaner looking code later down the road.